packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -273,11 +272,13 @@ const Select = <
               />
             )}
             {option.text}
-            {internalSelectedValues === option.value && (
-              <ListItemSecondaryAction>
-                <CheckIcon />
-              </ListItemSecondaryAction>
-            )}
+            {!hasMultipleChoices &&
+              (internalSelectedValues?.includes(option.value) ||
+                internalSelectedValues === option.value) && (
+                <ListItemSecondaryAction>
+                  <CheckIcon />
+                </ListItemSecondaryAction>
+              )}
```

Comment:
This is fine. Makes sense based on Figma.

I would like to check with Design to see if we want a checkbox on the left and a checkmark on the right because it's a bit jarring, the difference between the two side-by-side, but when you can only see one at a time, it's probably fine?

I still want to get their opinion now that we've seen this.